infrastructure/kubernetes/monitoring/kustomization.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - namespace.yaml
+  - prometheus/persistent-volume-claim.yaml
+  - prometheus/configmap.yaml
+  - prometheus/deployment.yaml
+  - prometheus/service.yaml
+  - grafana/persistent-volume-claim.yaml
+  - grafana/deployment.yaml
+  - grafana/service.yaml
+  # - grafana/config.yaml
+  - scrapping/deployment.yaml
+  - scrapping/service.yaml
+  - scrapping/configmap.yaml
+  - ingress.yaml
+
+labels:
+  - includeSelectors: true
+    pairs:
+      app.kubernetes.io/part-of: mezo-monitoring
```

Let's strip the prefix.
```suggestion
      app.kubernetes.io/part-of: monitoring
```

---

addressed in 4a2043eecd31336045fb360218e633e2d4441c65